extensions/libarchive/MergeContent.cpp

```diff
@@ -156,7 +159,7 @@ void MergeContent::onSchedule(core::ProcessContext *context, core::ProcessSessio
         this->demarcatorContent_ = readContent(demarcator_);
     }
   }
-  if (delimiterStratgey_ == DELIMITER_STRATEGY_TEXT) {
+  if (delimiterStrategy_ == DELIMITER_STRATEGY_TEXT) {
     this->headerContent_ = header_;
     this->footerContent_ = footer_;
```

Comment:
as we picked the "this-elimination" route, we should remove `this->` everywhere in this file